pubspec.yaml

```diff
@@ -12,6 +12,7 @@ dependencies:
   flip_card: ^0.2.1
   rxdart: ^0.21.0
   flutter_graphql: ^1.0.0-rc.3
+  firebase_auth: ^0.7.0
```

Comment:
![](https://media.tenor.com/images/fd5c410af4801a1159acf165cae15671/tenor.gif)

Reply:
Sacrifices were made. However, I applaud your choice of gif.

Reply:
I choose what fits the context.